client/less/challenge.less

```diff
@@ -138,7 +138,7 @@
 }
 
 .refresh-icon {
-  color: darkgreen;
+  color: #575757;
```

Comment:
@raisedadead 

> please declare this as a variable and use here.

Is there a preferred file I should declare the variable or can I do it in the challenge.less file?